tests/modules/NotificationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ *
+ * NotificationTest.php is unit test file for Notification module (modules/Notification.php).
+ *
+ */
+namespace Tests\FirstAide;
+$APPLICATION_DIR = str_replace('tests/modules', '', dirname(__FILE__));
+require_once $APPLICATION_DIR.'/includes/settings.php';
+
+/**
+ * Importing required modules/wrappers
+ */
+use FirstAide\Notification;
+use PHPUnit\Framework\TestCase;
+
+class NotificationTest extends TestCase
```

Comment:
@MehaKaushik According to me, these tests cover all the methods of the notification module. Therefore, their coverage is 100%. Do you have any other test suggestion?